packages/neutrino/bin/build.js

```diff
@@ -7,10 +7,7 @@ module.exports = (middleware, options) => {
   return build(middleware, options)
     .fork((errors) => {
       spinner.fail('Building project failed');
-      errors.forEach((err) => {
-        console.error(err.stack || err);
-        err.details && console.error(err.details);
-      });
+      errors.forEach(err => console.error(err));
```

Comment:
Ah yes, always forget that. You can solve that by brining in `unary` from ramda and putting console.error in it:

```js
const { unary } = require('ramda');

errors.forEach(unary(console.error));
```